src/app-layout/mobile-toolbar/index.tsx

```diff
@@ -88,6 +92,10 @@ export function MobileToolbar({
       document.body.classList.remove(styles['block-body-scroll']);
     }
   }, [anyPanelOpen]);
+
+  const splitNum = drawers?.items.length === 3 ? 3 : 2;
```

Comment:
Good catch. I think this logic should go inside the `splitItems` function, because it is the same for all cases

Also, it would be very convenient to test via `src/app-layout/drawer/__tests__/drawers-helpers.test.ts` setup

Reply:
It's only the case for mobile, not all. But I can add an `isMobile` argument to the function.

Reply:
Why is this mobile only? I think it makes sense for the desktop too. Every time there is a dropdown with just one item, it could be rendered without the overflow

Reply:
This is already the case! I added it as a feature to the [split items helper](https://github.com/cloudscape-design/components/pull/1416/files#diff-a0890a9725d3f94a60ee2a7aeed898f9771db07811a439e6313e92aa1bbdbe93R14).

Reply:
I see a code simplification opportunity. I will follow up when all currently planned features added and the dust settles